src/OutputManager.ts

```diff
@@ -81,7 +85,13 @@ export class OutputManager {
                 .filter(line => !ignoreEmpty || line.trim().length > 0)
                 .join("\n");
         }
-        return `<span class="${className}">${escapeHTML(spanText)}</span>`;
+        // If the HTML can be interpreted as colored, then render it that way
+        const coloredText = this.ansiUp.ansi_to_html(spanText);
+        if (coloredText !== spanText) {
+            return coloredText;
+        } else {
+            return (`<span class="${className}">${escapeHTML(spanText)}</span>`);
+        }
```

Comment:
```suggestion
        return `<span class="${className}">${this.ansiUp.ansi_to_html(spanText)}</span>`;
```

Reply:
Does ansiUp apply escaping to text that it doesn't modify?

Reply:
Seems to be the case, so I'll remove the if